public/styles.css

```diff
@@ -14,3 +14,8 @@ body {
   line-height: 27px;
   color: #091133;
 }
+
+#__next {
+  width: calc(100vw - (100vw - 100%));
+  overflow-x: hidden;
+}
```

Comment:
We could do that to the div inside `index.tsx`, but since `IndexPage` is a grandchild of `body` it would not inherit the natural scrolling behavior from browsers.
For example on desktop users using the keyboard to scroll would first have to click on the page to start scrolling, and probably on mobile the top-bar (the one with the address/URL) would stay visible until we reach the bottom of the page, and not hide when users scroll down.